package.json

```diff
@@ -69,6 +69,8 @@
     "eslint-plugin-vue": "^8.0.3",
     "jest": "^26.6.3",
     "minimist": "^1.2.6",
+    "pdf-to-excel": "^0.3.2",
+    "pdfjs-dist": "^3.4.120",
```

Comment:
Do we use this dependency anywhere? If not, can we delete it?

Reply:
We use pdf-to-excel but not pdfjs-dist so I removed that